packages/atomic/src/components/search/atomic-result-template/atomic-result-template.mdx

```diff
@@ -0,0 +1,118 @@
+ * **Note:** Any `<script>` tags that are defined inside a `<template>` element will not be executed when the results are being rendered.
+
+Example using the `atomic-result-list`:
+```html
+<atomic-result-list>
+  <atomic-result-template>
+    <template>
+      <atomic-result-section-visual>
+        <atomic-result-image field="ytthumbnailurl" fallback="https://picsum.photos/350"></atomic-result-image>
+      </atomic-result-section-visual>
+      
+      <atomic-result-section-badges>
+        <atomic-field-condition must-match-sourcetype="YouTube">
+          <atomic-result-badge
+            label="YouTube"
+            class="youtube-badge"
+          ></atomic-result-badge>
+        </atomic-field-condition>
+        
+        <atomic-result-badge
+          icon="https://raw.githubusercontent.com/Rush/Font-Awesome-SVG-PNG/master/black/svg/language.svg"
+        >
+          <atomic-result-multi-value-text
+            field="language"
+          ></atomic-result-multi-value-text>
+        </atomic-result-badge>
+      </atomic-result-section-badges>
+
+      <atomic-result-section-title>
+        <atomic-result-link></atomic-result-link>
+      </atomic-result-section-title>
+
+      <atomic-result-section-excerpt>
+        <atomic-result-text field="excerpt"></atomic-result-text>
+      </atomic-result-section-excerpt>
+
+      <atomic-result-section-bottom-metadata>
+        <atomic-result-fields-list>
+          <atomic-field-condition class="field" if-defined="source">
+            <span class="field-label">
+              <atomic-text value="source"></atomic-text>:
+            </span>
+            <atomic-result-text field="source"></atomic-result-text>
+          </atomic-field-condition>
+          
+          <atomic-field-condition class="field" if-defined="author">
+            <span class="field-label">
+              <atomic-text value="author"></atomic-text>:
+            </span>
+            <atomic-result-text field="author"></atomic-result-text>
+          </atomic-field-condition>
+          
+          <atomic-field-condition class="field" if-defined="date">
+            <span class="field-label">
+              <atomic-text value="date"></atomic-text>:
+            </span>
+            <atomic-result-date></atomic-result-date>
+          </atomic-field-condition>
+        </atomic-result-fields-list>
+      </atomic-result-section-bottom-metadata>
+    </template>
+  </atomic-result-template>
+</atomic-result-list>
+```
+
+## Template Conditions
+
+You can use conditions to display different templates based on result properties. **The order of declaration matters**: the first template whose conditions are met will be applied. If a default template (without conditions) is declared first, it will apply to all results, even if other templates with conditions are declared later.
```

Comment:
The plural "conditions" used here seems to imply that we can specify multiple conditions on a template, which I'm not sure is the case.

Can you validate and update the docs accordingly if needed?